Vagrantfile

```diff
@@ -24,6 +24,15 @@ Vagrant.configure(2) do |config|
   vm_box_version = ENV.fetch('SCF_VM_BOX_VERSION', ENV.fetch('VM_BOX_VERSION', '2.0.15'))
   vm_registry_mirror = ENV.fetch('SCF_VM_REGISTRY_MIRROR', ENV.fetch('VM_REGISTRY_MIRROR', ''))
 
+  HOME = "/home/vagrant"
+
+  # Set this environment variable to a directory containing shell scripts to be executed as part of
+  # the provisioning of the Vagrant machine.
+  custom_config_scripts_env = "CUSTOM_CONFIG_SCRIPTS"
```

Comment:
Can we use the `SCF_VM_` prefix for consistency with the other env variables?

Reply:
`SCF_VM_CUSTOM_SCRIPTS`?

Reply:
I mostly care about the prefix, but if you want to bike-shed the rest of the name, then I would prefer to include `CONFIG` or `SETUP` to make it clearer that these are one-time config scripts, and not additional private scripts that you just want to make available inside the VM.

Reply:
`SCF_VM_CUSTOM_SETUP_SCRIPTS` it is.